src/main/java/frc/robot/commands/SimpleControl.java

```diff
@@ -0,0 +1,189 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import com.chaos131.auto.ParsedCommand;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.commands.auto.AutoUtil;
+import frc.robot.subsystems.Feeder;
+import frc.robot.subsystems.Intake;
+import frc.robot.subsystems.Launcher;
+import frc.robot.subsystems.Lift;
+
+/**
+ * A class for controlling the mechanisms (intake, feeder, and flywheels) in a simple way. Each respective subsystem can optionally be added via chaining:
+ * 
+ * <code>
+ * new SimpleControl().intake(m_intake, 0.5).feeder(m_feeder, 0.3);
+ * </code
+ */
+public class SimpleControl extends Command {
```

Comment:
I added this class to make it easier to add these controls in RobotContainer.

In the future, this class should hopefully not be needed after we have our more complex commands that will also control height and tilt automatically.

Reply:
I then got carried away and added the ability to call this class easily from the auto script...